.github/workflows/sync-to-develop.yml

```diff
@@ -9,15 +9,36 @@ jobs:
   sync-to-develop:
     runs-on: ubuntu-latest
     steps:
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.19.1
       - name: Checkout
-        uses: actions/checkout@v1
-
-      - name: Merge from master to develop
-        uses: wingkwong/gh-action-nightly-merge@master
+        uses: actions/checkout@v3
         with:
-          stable_branch: 'master'
-          development_branch: 'develop'
-          allow_ff: true
-          allow_forks: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: nocodb
+      - name: Prepare GH Cli
+        id: gh
+        run: |
+          wget https://github.com/cli/cli/releases/download/v2.33.0/gh_2.33.0_linux_amd64.tar.gz
+          tar -xvf gh_2.33.0_linux_amd64.tar.gz
+          GH=gh_2.33.0_linux_amd64/bin/gh
+          echo "GH=${GH}" >> $GITHUB_OUTPUT
+      - name: Sync Master to Develop
+        id: sync
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd nocodb
+          git fetch --quiet --unshallow origin
+          git checkout master
+          BRANCH_NAME=oss/$(git rev-list -n 3 HEAD | tail -1 | cut -c1-8)
```

Comment:
suggest to add a comment on `git rev-list -n 3 HEAD | tail -1 | cut -c1-8`

Reply:
I've added a comment and also changed the logic to use last commit hash which is simpler, thanks